### PR TITLE
Feature liquibase setup

### PR DESCRIPTION
**Pull request by maksigors**

### CODE REVIEW

File: src/main/resources/db/changelog/changes/create-table-cars.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+--liquibase formatted sql
+--changeset igor_maks:create-table-cars splitStatements:true endDelimeter: ;
+CREATE TABLE IF NOT EXISTS `cars`  (
+                         `id` BIGINT(0) UNSIGNED NOT NULL AUTO_INCREMENT,
+                         `model` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
+                         `brand` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
+                         `type` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
+                         `inventory` BIGINT(0) UNSIGNED NOT NULL,
+                         `daily_fee` DECIMAL(5, 2) UNSIGNED NOT NULL,
+                         `is_deleted` BIT(1) NOT NULL DEFAULT b'0',
```
Comment (Gamemod13) [inline src/main/resources/db/changelog/changes/create-table-cars.sql:10]: remove
```suggestion
                         
```

File: src/main/resources/db/changelog/changes/create-table-payments.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+--changeset igor_maks:create-table-users splitStatements:true endDelimeter: ;
+
+CREATE TABLE IF NOT EXISTS `payments`  (
+                         `id` BIGINT(0) UNSIGNED NOT NULL AUTO_INCREMENT,
+                         `status` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
+                         `type` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
+						`rental_id` BIGINT unsigned NOT NULL,
+                         `session_url` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
+                         `session_id` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
+                         `suma` DECIMAL(5, 2) UNSIGNED NOT NULL,
```
Comment (Gamemod13) [inline src/main/resources/db/changelog/changes/create-table-payments.sql:11]:
```suggestion
                         `amount_to_pay` DECIMAL(5, 2) UNSIGNED NOT NULL,
```

File: src/main/resources/db/changelog/changes/create-table-payments.sql [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+                         `is_deleted` BIT(1) NOT NULL DEFAULT b'0',
```
Comment (Gamemod13) [inline src/main/resources/db/changelog/changes/create-table-payments.sql:12]:
```suggestion
                         
```

File: src/main/resources/db/changelog/changes/create-table-payments.sql [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+                         PRIMARY KEY (`id`) USING BTREE
```
Comment (Gamemod13) [inline src/main/resources/db/changelog/changes/create-table-payments.sql:13]: Add rental relation (FK)

File: src/main/resources/db/changelog/changes/create-table-rentals.sql [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+--changeset igor_maks:create-table-users splitStatements:true endDelimeter: ;
+
+CREATE TABLE IF NOT EXISTS `rentals`  (
+                         `id` BIGINT(0) UNSIGNED NOT NULL AUTO_INCREMENT,
+						 `rental_date` DATETIME NOT NULL,
+						 `return_date` DATETIME NOT NULL,
+						 `actual_return_date` DATETIME NOT NULL,
+						`car_id` BIGINT unsigned NOT NULL,
+						`user_id` BIGINT unsigned NOT NULL,
+                         `is_deleted` BIT(1) NOT NULL DEFAULT b'0',
```
Comment (Gamemod13) [inline src/main/resources/db/changelog/changes/create-table-rentals.sql:11]:
```suggestion
                       
```

Comment (Gamemod13) [inline src/main/resources/db/changelog/changes/create-table-rentals.sql:11]: Add references for User and Car

File: src/main/resources/db/changelog/changes/create-table-users.sql [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+--changeset igor_maks:create-table-users splitStatements:true endDelimeter: ;
+
+CREATE TABLE IF NOT EXISTS `users`  (
+                         `id` BIGINT(0) UNSIGNED NOT NULL AUTO_INCREMENT,
+                         `email` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
+                         `first_name` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
+                         `last_name` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
+                         `password` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
+                         `role` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
+                         `is_deleted` BIT(1) NOT NULL DEFAULT b'0',
```
Comment (Gamemod13) [inline src/main/resources/db/changelog/changes/create-table-users.sql:11]:
```suggestion
                         
```

File: src/main/resources/db/changelog/changes/create-table-users.sql [hunk 7]
```diff
@@ -0,0 +1,13 @@
+--liquibase formatted sql
+--changeset igor_maks:create-table-users splitStatements:true endDelimeter: ;
+
+CREATE TABLE IF NOT EXISTS `users`  (
+                         `id` BIGINT(0) UNSIGNED NOT NULL AUTO_INCREMENT,
+                         `email` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
+                         `first_name` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
+                         `last_name` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
+                         `password` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
+                         `role` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
```
Comment (Gamemod13) [inline src/main/resources/db/changelog/changes/create-table-users.sql:10]:
```suggestion
                         `role` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL DEFAULT 'CUSTOMER',
```

**Review by Gamemod13 — APPROVED** [review]

**Review by Tupadzaky — APPROVED** [review]

**Pull request merged**